<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLViewer2</class>
 <widget class="QMainWindow" name="PCLViewer2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>167</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="captureImageButton">
       <property name="text">
        <string>Capture Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getFloorButton">
       <property name="text">
        <string>Detect Floor Plane</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findClusterButton">
       <property name="text">
        <string>Find Clusters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getDistanceButton">
       <property name="text">
        <string>Move Claw to Object</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textDisplay">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="floorAlphaSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPCL_Viewer">
    <property name="title">
     <string>PCL Viewer</string>
    </property>
   </widget>
   <addaction name="menuPCL_Viewer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
